cmd/crio/main.go

```diff
@@ -413,7 +413,7 @@ func main() {
 		}()
 
 		go func() {
-			err := service.StartInfoEndpoints()
+			err := service.StartInfoEndpoints(lis)
```

Comment:
this should probably go after `s.Serve(lis)` below at line 422 cause the listener isn't initialized maybe